DotNet/emitter.js

```diff
@@ -140,12 +141,12 @@
 		let level = `${test.config.defaultMatchSettings.accessibilitySettings.level}`
 		let version = `${test.config.defaultMatchSettings.accessibilitySettings.guidelinesVersion}`
 		addHook('beforeEach', dot_net`AccessibilitySettings settings = new AccessibilitySettings(AccessibilityLevel.` + level + `, AccessibilityGuidelinesVersion.` + version + `);
-        Applitools.Selenium.Configuration configuration = eyes.GetConfiguration();
+        var configuration = eyes.GetConfiguration();
         configuration.SetAccessibilityValidation(settings);
         eyes.SetConfiguration(configuration);`)
 	}
 	if ("browsersInfo" in test.config) {
-		addHook('beforeEach', dot_net`Applitools.Selenium.IConfiguration config = eyes.GetConfiguration();`)
+		addHook('beforeEach', dot_net`var config = eyes.GetConfiguration();`)
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
		addHook("beforeEach", dot_net`var config = eyes.GetConfiguration();`)
```